package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "kilt-extension",
+  "name": "kilt-extension-api",
```

Comment:
I thought the first version was already published with a different name

Reply:
I'd rather call it kilt-extension-api since it's the API for KILT extensions. `kilt-extension` sounds like it's an extension itself.
It's anyways not used by anyone at the moment. So i would think that this is fine.